src/interfaces/IPoolManager.sol

```diff
@@ -151,19 +152,29 @@ interface IPoolManager is IProtocolFees, IERC6909Claims, IExtsload, IExttload {
         external
         returns (BalanceDelta);
 
+    /// @notice Writes the current ERC20 balance of the specified currency to transient storage
+    /// This is used to checkpoint balances for the manager and derive deltas for the caller.
+    /// @dev This MUST be called before any ERC20 tokens are sent into the contract.
+    function sync(Currency currency) external returns (uint256 balance);
+
     /// @notice Called by the user to net out some value owed to the user
     /// @dev Can also be used as a mechanism for _free_ flash loans
     function take(Currency currency, address to, uint256 amount) external;
 
+    /// @notice Called by the user to pay what is owed
+    function settle(Currency currency) external payable returns (uint256 paid);
+
+    /// @notice Called to clear a positive balance without a corresponding transfer.
```

Comment:
Honestly i think this needs to be more severe in its warning. Imo it needs to state
```
Any currency that is cleared, will be non-retreivable, and locked in the contract permanently
```

Or something like that. I wont want anyone ever using this function thinking theyre clearing a debt or something by accident.